pkg/crc/api/types.go

```diff
@@ -25,6 +26,20 @@ type clusterOpsRequest struct {
 
 // commandRequest struct is used to decode the json request from tray
 type commandRequest struct {
-	Command string            `json:"command"`
-	Args    map[string]string `json:"args,omitempty"`
+	Command string `json:"command"`
+	Args    json.RawMessage
+}
+
+// setOrUnsetConfigResult struct is used to return the result of
+// setconfig/unsetconfig command
+type setOrUnsetConfigResult struct {
+	Error   string
+	Success bool
```

Comment:
Imo it's a mistake that some crc APIs return both an error and a success boolean. If `Success` is true, then `Error` must be `""`, if `Success` is false, then `Error` must be  non-empty, so they are mutually exclusive, so I would either send a message with only an `Error` field, or a message with the command result. No need for `Success`, unless maybe if the response would be empty otherwise.

I wonder if set/unset should return the list of config values it set/unset?

Reply:
Yes they are mutually exclusive, only the error would be enough to know if succeeded or failed but for `setconfig` and `unsetconfig` there is no command result, so `Success` is used.

> I wonder if set/unset should return the list of config values it set/unset?

I thought about this, in case any of the set/unset on the provided keys fails, we are getting back an error (err msg contains the name of the keys but harder to extract)